### ISSUE TITLE
Add justjson as an alternative json solution.

### ISSUE THREAD

**Issue by kvinwang** [Phala-Network/phala-wiki-next#195]
Looks like It won't produce unnecessary fp instructions: https://github.com/khonsulabs/justjson/

And maybe another way: making our own simple JSON parser based on https://github.com/zesterer/chumsky/blob/master/examples/json.rs.

**Comment by h4x3rotab** [comment]
Looks nice. Do you think it's better than having our own serde json in our own crates (e.g. pink-web3)?

**Comment by kvinwang** [comment]
> Do you think it's better than having our own serde json in our own crates (e.g. pink-web3)?

No, `justjson` doesn't support serde. It's not easy to replace serde_json with it in pink-web3.

It can be a lightweight alternative to some simple scenarios.
